### PR TITLE
add token balances scan

### PR DESCRIPTION
**Pull request by tarasherasymchuk**
Added token balances scan. A `TrustRelationship` entity of this structure is created and pushed to UTU:
```
{
    "type": "erc20_balance",
    "sourceCriteria": {
        "type": "wallet",
        "name": "0x...",
        "ids": {
            "address": "0x..."
        }
    },
    "targetCriteria": {
        "type": "erc20",
        "name": "USDT",
        "ids": {
            "address": "0x..."
        }
    },
    "properties": {
        "balance": "30"
    }
}
```

### CODE REVIEW

**Comment by databu** [comment]
Hi @tarasherasymchuk , a few things to change:
* type of the relationship should be `"has"` instead of `"erc20_balance"`
* type of source criteria should be `"address"` instead of `"wallet"`

**Comment by tarasherasymchuk** [comment]
@databu done

waiting for the review from @musayann

**Review by musayann — APPROVED** [review]

**Comment by musayann** [comment]
Looks good to me

**Pull request merged**
